src/EventStore.ClusterNode.Web/styles/app.css

```
@@ -0,0 +1,15 @@
+@import "tailwindcss" source(none);
+
+@source "../clusternode-web/index.html";
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@source" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)